man/systemd.network.xml

```diff
@@ -2570,7 +2570,9 @@ NFTSet=prefix:netdev:filter:eth_ipv4_prefix</programlisting>
           effect of the <option>Domains=</option> setting. If set to <option>route</option>, the domain name
           received from the DHCP server will be used for routing DNS queries only, but not for searching,
           similarly to the effect of the <option>Domains=</option> setting when the argument is prefixed with
-          <literal>~</literal>. Defaults to false.</para>
+          <literal>~</literal>. When unspecified, the value specified in the same setting in
+          <citerefentry><refentrytitle>networkd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+          which defaults to <literal>no</literal>, will be used.</para>
```

Comment:
Don't include what the default value of the manager's parameter is; that is redundant and also could easily be missed by downstream distros who change the default value of the manager's parameter (and only change the doc for `networkd.conf`). Here, just state that the default value (of the network's parameter) is the value of the manager's parameter (i.e. leave out `which defaults to no`)